src/index.js

```diff
@@ -79,31 +78,29 @@ const createCaseSensitiveHook = require('./create_case_sensitive_hook');
  */
 
 module.exports = async (options) => {
-    const opts = _.defaults({}, options, REQUEST_DEFAULT_OPTIONS);
-
     const {
         url,
-        method = 'GET',
-        headers = {},
-        followRedirect = true,
-        maxRedirects = 20,
-        throwOnHttpErrors = false,
-        abortFunction = null,
-        timeoutSecs = 30,
-        ignoreSslErrors = false,
-        decodeBody = true,
-        json = false,
-        stream = false,
-        useBrotli = false,
+        method = REQUEST_DEFAULT_OPTIONS.method,
+        headers = REQUEST_DEFAULT_OPTIONS.headers,
+        followRedirect = REQUEST_DEFAULT_OPTIONS.followRedirect,
+        maxRedirects = REQUEST_DEFAULT_OPTIONS.maxRedirects,
+        throwOnHttpErrors = REQUEST_DEFAULT_OPTIONS.throwHttpErrors,
+        abortFunction = REQUEST_DEFAULT_OPTIONS.abortFunction,
+        timeoutSecs = REQUEST_DEFAULT_OPTIONS.timeoutSecs,
+        ignoreSslErrors = REQUEST_DEFAULT_OPTIONS.ignoreSslErrors,
+        decodeBody = REQUEST_DEFAULT_OPTIONS.decodeBody,
+        json = REQUEST_DEFAULT_OPTIONS.json,
+        stream = REQUEST_DEFAULT_OPTIONS.stream,
+        useBrotli = REQUEST_DEFAULT_OPTIONS.useBrotli,
```

Comment:
This is unnecessarily complicated. Just delete the constants file and move the default values directly here. Unless you need them elsewhere?

Reply:
Yes, I need them in the tests...

Reply:
It kinda kills the point of having the defaults readable where they're used though :D